app/Server.hs

```diff
@@ -23,6 +23,8 @@ import FacebookUtilities
 import Config (markdownPath, serverConf)
 import qualified Data.Text.Lazy.IO as LazyIO
 import Data.Int (Int64)
+<<<<<<< HEAD
```

Comment:
This is a remnant of a merge conflict, and should be removed. (Always check for these before committing your work.)